monitoring/kafka-connect/dashboard.py

```diff
@@ -0,0 +1,820 @@
+
+    source_task_source_record_poll_rate = metrics.Metric(
+        'kafka_connect_source_task_source_record_poll_rate', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    source_task_source_record_write_rate = metrics.Metric(
+        'kafka_connect_source_task_source_record_write_rate', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    source_task_source_record_active_count = metrics.Metric(
+        'kafka_connect_source_task_source_record_active_count', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    source_task_source_record_active_count_avg = metrics.Metric(
+        'kafka_connect_source_task_source_record_active_count_avg', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    source_task_source_record_active_count_max = metrics.Metric(
+        'kafka_connect_source_task_source_record_active_count_max', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    sink_task_partition_count = metrics.Metric(
+        'kafka_connect_sink_task_partition_count', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')  
+
+    sink_task_put_batch_avg_time_ms = metrics.Metric(
+        'kafka_connect_sink_task_put_batch_avg_time_ms', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    sink_task_put_batch_max_time_ms = metrics.Metric(
+        'kafka_connect_sink_task_put_batch_max_time_ms', 'instance', 'connector', 'task', job='${job}'
+    ).with_defaults('instance=~"$instance"', 'connector=~"$connector"', 'task!=""')
+
+    process_cpu_seconds_total = metrics.CounterMetric(
+        'process_cpu_seconds_total', 'instance', job='${job}'
+    ).with_defaults('instance=~"$instance"')
+
+    jvm_memory_bytes_used = metrics.Metric(
+        'jvm_memory_bytes_used', 'area', 'instance', job='${job}'
+    ).with_defaults('instance=~"$instance"')
+
+    jvm_memory_bytes_max = metrics.Metric(
+        'jvm_memory_bytes_max', 'area', 'instance', job='${job}'
+    ).with_defaults('instance=~"$instance"')
+
+    jvm_gc_collection_seconds = metrics.BucketMetric(
+        'jvm_gc_collection_seconds', 'gc', 'instance', job='${job}'
+    ).with_defaults('instance=~"$instance"')
+
+
+up = Stat(
+    title="Workers",
+    description="Number of workers available",
+    dataSource='${DS_PROMETHEUS}',
+    minValue='0',
+    maxValue='${replicas}',
+    reduceCalc='last',
+    targets=[Target(
+        expr='sum(up{job="${job}"})',
```

Comment:
```suggestion
        expr='sum(up{job="${job}", namespace="${namespace}"})',
```

Reply:
Not sure we should filter on 'instance' here, as it would be displayed as a warning (yellow/red)...